sdk/storage/storage-blob-changefeed/review/storage-blob-change-feed.api.md

```diff
@@ -0,0 +1,91 @@
+## API Report File for "@azure/storage-blob-changefeed"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { BlobServiceClient } from '@azure/storage-blob';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+
+// @public (undocumented)
+export class BlobChangeFeedClient {
+    constructor(blobServiceClient: BlobServiceClient);
+    // (undocumented)
+    getChanges(options?: ChangeFeedGetChangesOptions): PagedAsyncIterableIterator<BlobChangeFeedEvent, BlobChangeFeedEventPage>;
+    }
+
+// @public (undocumented)
+export interface BlobChangeFeedEvent {
+    // (undocumented)
+    data: BlobChangeFeedEventData;
+    // (undocumented)
+    dataVersion?: string;
+    // (undocumented)
+    eventTime: string;
```

Comment:
Should this be lifted to a date? It's strange I have to pass in dates via ChangeFeedGetChangesOptions and the time I'm filtering on is a string.

Reply:
Changed to `Date`. Using string as the raw data is a Date string.